atlasdb-client/src/main/java/com/palantir/atlasdb/logging/LoggingArgs.java

```diff
@@ -82,13 +83,28 @@ public static SafeAndUnsafeTableReferences tableRefs(Collection<TableReference>
                 .build();
     }
 
+    public static List<String> tableRefValues(Collection<TableReference> tables) {
+        return tables.stream().map(LoggingArgs::tableRefValue).collect(Collectors.toList());
```

Comment:
We should try to avoid the allocation overhead if we're not tracing by lazily transforming only as needed
```
    public static Iterable<TableReference> tableRefValues(Iterable<TableReference> tables) {
        return Iterables.transform(tables, LoggingArgs::tableRefValue);
    }
```